action.yml

```diff
@@ -12,4 +12,4 @@ outputs:
 
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  image: 'docker://ghcr.io/paritytech/get-fellows-action/action:1.0.0'
```

Comment:
just curious: is there a way
to have here something like "latest" alias and only change version of package json during release?
so also the release would be simple action of 'yarn version <patch/minor/major>'
or you wanted the release process always be so specific through PR?